content/department-processes/team-onboarding/_index.md

```diff
@@ -57,7 +57,13 @@ sheena.oconnell@umuzi.org
 #### Regular meetings: 
 Daily stand ups with recruits at 9am. 
 
-Daily tech meeting / check-in with tech facilitators and managers at 3:30pm on Mondays and 4pm every other day.
+Daily tech meeting / check-in with tech facilitators and managers at 3:30pm on Mondays and 4pm every other day. Tech team daily meeting format:
+
+ - Mondays: Review on the weekly proceedings
```

Comment:
On a monday meeting we review the prototyping that we are doing. This is recorded and the results and learning are recorded as this data is useful in the daily running and future prototyping.